cmd/crypto/header.go

```diff
@@ -95,13 +97,13 @@ type ssecCopy struct{}
 // IsRequested returns true if the HTTP headers contains
 // at least one SSE-C header. SSE-C copy headers are ignored.
 func (ssec) IsRequested(h http.Header) bool {
-	if _, ok := h[SSECAlgorithm]; ok {
+	if h.Get(SSECAlgorithm) != "" {
```

Comment:
Not correct. The header can be set but be empty.

Reply:
fixed.